google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/util/AppEngineUtil.java

```diff
@@ -159,8 +159,8 @@ private static boolean doesArtifactMatchEnvironment(Artifact artifact,
     return moduleDeploymentSources;
   }
 
-  public static void setupAppEngineArtifactCombobox(@NotNull Project project,
-      final @NotNull JComboBox comboBox, final boolean withAppEngineFacetOnly) {
+  public static void setupAppEngineStandardArtifactCombobox(@NotNull Project project,
```

Comment:
nit: Can you run the format tool on this? It should put both parameters on the following line